wandb/sdk/wandb_run.py

```diff
@@ -2040,6 +2040,12 @@ def _on_ready(self) -> None:
         self._freeze()
 
     def _log_job(self) -> None:
+        # don't produce a job if the run is sourced from a job
```

Comment:
interesting, are any existing launch paths that were automatically creating jobs now going to be silent?

Reply:
Before it could happen that if you had 2 versions of the same job, and you re ran them switching back and forth you'd just endlessly make new jobs. Now you wont. Instead we'll just mark this run as a user of that job